cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs

```diff
@@ -1527,9 +1527,8 @@ runQueryCommitteeState
   eraInMode <- toEraInMode cEra cMode
     & hoistMaybe (QueryCmdEraConsensusModeMismatch (AnyConsensusMode cMode) (AnyCardanoEra cEra))
 
-  committeeState <- runQuery localNodeConnInfo $ queryCommitteeState eraInMode sbe
-  writeOutput mOutFile $
-    Map.assocs $ committeeState ^. Ledger.csCommitteeCredsL
+  committeeState <- runQuery localNodeConnInfo $ queryCommitteeMembersState eraInMode sbe Set.empty Set.empty Set.empty
+  liftIO . putStr $ show committeeState
```

Comment:
What's in the committee state?

Would it be useful to be able to write it to a file?

Also wondering if it's worth having a JSON encoding of it for consistency?

Reply:
You're right,  no reason to not write it in the output file as it was before.  I will change it.

The result of the query  ([CommitteeMembersState](https://github.com/input-output-hk/cardano-ledger/blob/master/libs/cardano-ledger-api/src/Cardano/Ledger/Api/State/Query/CommitteeMembersState.hs#L141)) is a more sophisticated version of CommitteeState, containing more info about the members, such as status, expiry, hot credential status.  Basically it's putting together information about committee members, from both sources: Committee (which is the current committee cold creds)  and CommitteState  (which is the mapping of cold to hot credentials of committee members).

> Also wondering if it's worth having a JSON encoding of it for consistency?

The type representing the result of this   query (CommitteeMembersState) [does have a JSON encoding defined in ledger. ](https://github.com/input-output-hk/cardano-ledger/blob/master/libs/cardano-ledger-api/src/Cardano/Ledger/Api/State/Query/CommitteeMembersState.hs#L165)
So I suppose your suggestion is to use it, but could you explain where? Is there an example of this?

Reply:
Updated to write it as json.